### PR TITLE
Added Owoify command

### LINKED ISSUE #11: Owoify command

### PR DESCRIPTION
**Pull request by MorphTuple**
# A merge-able PR

Closes #11 

Why are you not uploading the file?
This way we only need to delete the file and restart the bot if we ever make changes to it that need hotfixing

### CODE REVIEW

File: src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Owoify.kt [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+import net.dv8tion.jda.api.utils.MarkdownSanitizer
+import okhttp3.Request
+import java.io.File
+import java.util.*
+
+class Owoify : AbstractCommand(
+    "owoify",
+    CommandCategory.FUN,
+    // TODO: Change this description
+    "Yuuto can turn owoify your text! (whatever that means)",
```
Comment (zsotroav) [inline src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Owoify.kt]: Well, either remove or do the ToDo.
I think this description is ok, but maybe having an owoified text would be better.

File: src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Owoify.kt [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    "owoify [level] <text>"
```
Comment (zsotroav) [inline src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Owoify.kt]: Describing that `[level]` is optional would be good, and having the levels here or somewhere easily visible would be helpful.

File: src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Owoify.kt [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+) {
+    private val levels = listOf("easy", "medium", "hard")
+
+    init {
+        downloadFile()
+    }
+
+    override fun run(args: MutableList<String>, event: GuildMessageReceivedEvent) {
+        if (args.size == 0) {
+            event.channel.sendMessage("Sorry, but you need to provide me a message to owoify").queue()
```
Comment (zsotroav) [inline src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Owoify.kt]: Inconsistent error messages, missing End of Sentence Punctuation: `.` or `!` (preferred)

File: src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Owoify.kt [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+import java.io.File
+import java.util.*
+
+class Owoify : AbstractCommand(
+    "owoify",
+    CommandCategory.FUN,
+    // TODO: Change this description
+    "Yuuto can turn owoify your text! (whatever that means)",
+    "owoify [level] <text>"
+) {
```
Comment (zsotroav) [inline src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Owoify.kt:38]: Add aliases, as having `owoify` as the only way to call the command might annoy some people.
```suggestion
) {
    override val aliases = arrayOf("owo", "uwu")

```

**Review by zsotroav — CHANGES_REQUESTED** [review]
I've encountered some Windows-specific (or something like that) during testing, so I can only judge it by looking at the code.

File: src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Owoify.kt (hunk 4 above)
Comment (duncte123) [reply src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Owoify.kt:38]: uwu doesn't make sense as an alias

**Review by zsotroav — APPROVED** [review]
Code looks good to me, the minimal testing that I've done worked.

**Review by MorphTuple — COMMENTED** [review]
LGTM~!

**Review by duncte123 — APPROVED** [review]

**Review by Leguu — APPROVED** [review]
Looks good.

**Pull request merged**